package.json

```diff
@@ -19,7 +19,7 @@
     "@types/react": "^16.0.5",
     "@types/react-dom": "^15.5.4",
     "@types/react-router-dom": "^4.0.7",
-    "material-ui": "next",
+    "material-ui": "^1.0.0-beta.10",
```

Comment:
I can double check if changing this back to `next` will work just as well, but if this file is going to change whenever we update the packages that we're using anyways, then I don't really see the point.